shell/packages/sandstorm-accounts-ui/login_buttons.js

```diff
@@ -5,6 +5,9 @@ var helpers = {
   isDemoUser: function () {
     return this._db.isDemoUser();
   },
+  isAdmin: function () {
+    return Meteor.user().isAdmin;
```

Comment:
To be more clear: in the longer future, we intend to have SandstormDb wrap all accesses to the collections, and that includes Meteor's internal collections, so we prefer to wrap that here.

It should make it easier for us to 1) mock out SandstormDb in unit tests and 2) eventually shift away from Mongo someday, as described in [The Great Frontend Refactor](https://oasis.sandstorm.io/grain/kj9fopjyT2CdqC3i7fZN2z).